src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -187,21 +170,6 @@ const AdminDashboard = () => {
     setConfirmStatus('pending');
   };
 
-  //   pagination functions
-  const handleNextClick = evt => {
-    evt.preventDefault();
-    if (pageNumber < lastPage) {
-      setPageNumber(pageNumber + 1);
-    }
```

Comment:
Similar to the changeSelected event handler above, these actually should be being used, right? Like when we want to travel to the next or previous page? I guess this functionality already exists, hence the removal of this unused code.

Reply:
There is a similar piece of code in this file that prevents page number from becoming higher than last page, the piece here would not have a real functionality what why it was not being used since the page number is always supposed to be less than last page. This piece of code would change the state of page number to greater than what it currently is so if it was working it would be conflicting. I may be wrong though. Any feedback would be appreciated 

`  const approveAndRejectHandler = evt => {
    evt.preventDefault();
    const [reviewedIncidents, unreviewedIncidents] = sortApproved(
      currList,
      selected
    );
    putIncidents(oktaAxios, reviewedIncidents, confirmStatus);
    if (listType === 'unapproved') {
      setUnapprovedIncidents(unreviewedIncidents);
    } else if (listType === 'form-responses') {
      //did 'else if' incase we wanted to add Reject button to Approved list
      setFormResponses(unreviewedIncidents);
    }
    setAllSelected(false);
    setSelected([]);
    setConfirmApprove(false);
    setConfirmReject(false);
    if (pageNumber > lastPage) {
      setPageNumber(pageNumber - 1);
    }
  };`

Reply:
ok that makes sense

Reply:
After looking at this code I became curious as to why the original author chose to  cover two different, if very similar, cases in two disparate pieces of code. Seems like a good catch to reduce redundancy as long as there isn't some unintended consequence.